### DISCUSSION TITLE
/Pay Command  ?

### DISCUSSION THREAD

**Discussion by Blueeyestar** [Zrips/CMI#2686] [Enhancement]
In the new 1.15.1 version there is currently no plugin that / Pay Command offers as with Essentials. It would be nice if Cmi could offer this too. I do not find another possibility. I would be happy if this can be installed

**Comment by mercurialmusic** [comment]
It's already a thing: /money pay [username] [amount]
If you don't want the /money prefix just alias it in CustomAlias.yml:
```
  pay:
    Cmds:
    - cmi money pay $$1-
```

**Comment by Blueeyestar** [comment]
No that was misunderstood. I would like if you enter 10 thaler as an example / spawn. or like in my case / repair vehicle that you get 20 taler deducted

**Comment by mercurialmusic** [comment]
So you want to charge for using certain commands? That could also be done using CustomAlias.yml, but with specialized commands. See: https://www.zrips.net/cmi/commands/specialized/ for info on those. Then in CustomAlias.yml you could use, for example:

```
  spawn:
    Cmds:
    - moneycost:[10]?! cmi spawn
```

The question mark is optional, but it would inform the player if they don't have enough money to perform the command -- otherwise it would fail silently.
